inference-engine/tests_deprecated/functional/cldnn/shared_tests_instance/common_single_layer_tests/single_layer_tests.cpp

```diff
@@ -101,10 +101,9 @@ getTestCaseName(testing::TestParamInfo<std::tuple<InitialShapes, NewShapes, Plug
     return "CLDNN" + helper->getType();
 }
 
-#if (defined INSTANTIATE_TESTS)
-
 INSTANTIATE_TEST_CASE_P(
-        Conv_nightly, CommonSingleLayerTest,
+        // TODO: rewrite to ngraph to have reshape functionality
+        DISABLED_Conv_nightly, CommonSingleLayerTest,
```

Comment:
Should we remove all old disabled tests?

Or please create a ticket to rewrite these tests on nGraph.

Reply:
Will create tickets